git-annex-remote-rclone

```diff
@@ -16,268 +14,175 @@
+            if not self.remote_prefix:
+                self.remote_prefix = "git-annex"
+            if self.remote_prefix == "/":
+                raise RuntimeError("storing objects directly in the root (/) is not supported")
+            set_config("prefix", self.remote_prefix)
+
+            self.remote_target = get_config("target")
+            set_config("target", self.remote_target)
+
+            self.rclone_layout = get_config("rclone_layout") or "lower"
+            self._validate_layout()
+            set_config("rclone_layout", self.rclone_layout)
+
+            if not self.remote_target:
+                raise RuntimeError("rclone remote target must be specified (use target=parameter)")
+
+            if rclone("mkdir", f"{self.remote_target}:{self.remote_prefix}")[0]:
+                raise RuntimeError("Failed to create directory on remote. Ensure that 'rclone config' has been run.")
+
+        except RuntimeError as err:
+            say("INITREMOTE-FAILURE", err)
+        else:
+            say("INITREMOTE-SUCCESS")
+
+    def prepare(self):
+        self.remote_prefix = get_config("prefix")
+        self.remote_target = get_config("target")
+        self.rclone_layout = get_config("rclone_layout") or "lower"
+        try:
+            self._validate_layout()
+        except RuntimeError as err:
+            say("PREPARE-FAILURE", err)
+        else:
+            say("PREPARE-SUCCESS")
+
+    def transfer(self, op, key, *file):
+        file = " ".join(file)
+        location = self._calculate_location(key)
+
+        if op == "STORE":
+            # Store the file to a location
+            # based on the key.
+            # XXX when at all possible, send PROGRESS
+            if not os.path.exists(file):
+                say("TRANSFER-FAILURE", "STORE", key, f"asked to store non-existent file {file}")
+            else:
+                if not rclone("copy", file, location)[0]:
+                    say("TRANSFER-SUCCESS", "STORE", key)
+                else:
+                    say("TRANSFER-FAILURE", "STORE", key)
+
+        elif op == "RETRIEVE":
+            # Retrieve from a location based on
+            # the key, outputting to the file.
+            # XXX when easy to do, send PROGRESS
+            try:
+                with tempfile.TemporaryDirectory(prefix="rclone-annex-tmp.") as tmp:
+                    if rclone("copy", location + key, tmp)[0]:
+                        raise RuntimeError("Failed to copy")
+                    shutil.move(os.path.join(tmp, key), file)
+            except (OSError, IOError, RuntimeError):
+                say("TRANSFER-FAILURE", "RETRIEVE", key)
+            else:
+                say("TRANSFER-SUCCESS", "RETRIEVE", key)
+        else:
+            raise RuntimeError(f"Unknown operation {op}")
+
+    def checkpresent(self, key):
+        location = self._calculate_location(key)
+        code, output = rclone("size", "--json", location + key)
```

Comment:
You didn't translate the following line:

https://github.com/git-annex-remote-rclone/git-annex-remote-rclone/blob/ed7487817ab09da3746fe269d0eadb915100a74c/git-annex-remote-rclone#L120

Reply:
Hmm yes that was a function in use, in only one place. It read to me like something left in after a debugging session.
I've restored it instead with a "debug" flag on the `rclone(...)` function.